### PR TITLE
Readme game instructions

### PR DESCRIPTION
**Pull request by sumancrest0001**
Milestone-4
Improved the readme file and added the instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,43 @@
...
+## LANGUAGE USED
+### Ruby
+
+## GETTING STARTED
+### Follow the following instruction to install the game in you local machine.
+
+1. Open the git from the directory in which you want to install the game.
+2. Download all the related files using command 'git clone https://github.com/sumancrest0001/tic-tac-toe.git'
+3. After that, Navigate to the ruby-tic-tac-toe folder.
+4. Run a command 'bin/main.rb'
```
Comment (maelfosso) [inline README.md:13]: please don't you think it's rather `ruby bin/main.rb` ?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,43 @@
...
+1. First of all, one player chooses a mark (X or O). It will automatically assign remaining mark to another player.
+2. Player1, always starts the game and choose a mark.
+3. Next, Player2 chooses a mark from the remaining cells.
+4. If players try to choose already marked cells then it will show warning to the player and have to choose again.
+5. This process will continue till the game is end.
+6. You can play the game again by pressing "Y" and end the game with "N".
+7. It won't accept other than "Y" and "N".
+
+## CONTRIBUTOR
+sumancrest0001 (https://github.com/sumancrest0001)
```
Comment (maelfosso) [inline README.md:40]: Why Are you alone on that project ?

**Review by maelfosso — CHANGES_REQUESTED** [review]
Hello

Just one thing to correct before merging. Please indicate the correct way to start the game.
You could read the comment below.

Submit a new code review request after making the changes.
Thaks

File: README.md [hunk 3]
```diff
@@ -1,3 +1,43 @@
...
+## GETTING STARTED
+### Follow the following instruction to install the game in you local machine.
+
+1. Open the git from the directory in which you want to install the game.
+2. Download all the related files using command 'git clone https://github.com/sumancrest0001/tic-tac-toe.git'
+3. After that, Navigate to the ruby-tic-tac-toe folder.
+4. Run a command 'bin/main.rb'
+5. You can enjoy the game.
+
+## INSTRUCTION TO PLAYE THE GAME.
```
Comment (Ispirett) [inline README.md:16]: There is  a typo  in this line..

**Review by Ispirett — APPROVED** [review]
Great Job  @andynarf @sumancrest0001  continue the good work.

image
**You can merge this project now.** 

**Remember to place  descriptive commit messages.**

**Pull request merged**